src/formatDate.js

```diff
@@ -8,7 +8,44 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const splitSeparator = fromFormat[fromFormat.length - 1];
+  const splitDate = date.split(splitSeparator);
+
+  const joinSeparator = toFormat[toFormat.length - 1];
```

Comment:
Similar to the previous comment, the code assumes that the last element of `toFormat` is the separator. Ensure that this assumption holds true for all inputs or adjust the logic to handle different cases.